CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.7.0 (October 1, 2021)
+
+### Added
+
+-   New icons: EV Plug Type 1, EV Plug Type 2, EV Plug CCS Combo, EV Plug CHAdeMO, RFID
```

Comment:
Is the capitalization on chademo here the way we want it?

Reply:
Correct.